ts/Iterable.ts

```diff
@@ -42,39 +41,55 @@ class IterableConfig {
   * To view your existing integrations, navigate to Settings > Mobile Apps
   */
   pushIntegrationName?: string
+
   /**
   * When set to true, IterableSDK will automatically register and deregister notification tokens.
   */
   autoPushRegistration = true
+
   /**
   * When set to true, it will check for deferred deep links on first time app launch after installation from the App Store.
   */
   checkForDeferredDeeplink = false
+
   /**
   * How many seconds to wait before showing the next in-app, if there are more than one present
   */
   inAppDisplayInterval: number = 30.0
+
   /**
   * How many seconds to wait before showing the next in-app, if there are more than one present
   */
   urlHandler?: (url: string, context: IterableActionContext) => boolean
+
   /**
   * How to handle IterableActions which are other than 'openUrl'
   */
   customActionHandler?: (action: IterableAction, context: IterableActionContext) => boolean
+
   /**
   * Implement this protocol to override default in-app behavior.
   * By default, every single in-app will be shown as soon as it is available.
   * If more than 1 in-app is available, we show the first.
   */
   inAppHandler?: (message: IterableInAppMessage) => IterableInAppShowResponse
 
+  /**
+   * The handler with which your own calls to your backend containing the auth token happen
+   */
+  authTokenRequestedHandler?: () => Promise<string | undefined>
```

Comment:
This is called `authHandler`/`authDelegate` in native SDKs. Is there a reason to have a different name here?